### PR TITLE
Thumbnail previews in "All our charts" block

### PR DESCRIPTION
**Pull request by mlbrgl**
image

Intermediate step towards the fully-fledged solution described in #1380 (see rationale in [comment](https://github.com/owid/owid-grapher/issues/1380#issuecomment-1106426730))
- add thumbnails previews
- better chart switching UX on mobile
- block open by default

👉 [Preview](https://hans-owid.netlify.app/maternal-mortality)

### CODE REVIEW

**Comment by danielgavrilov** [comment]
Nice! Haven't done a code review yet, but some design thoughts: 

- The **active state was confusing** for me – I was wondering why the first chart is disabled. Instead of de-emphasising, maybe it's better to emphasise the selection? I think it's what I was expecting. 

- A bit hesitant on the benefit of **expanded by default**. As I was scrolling down, my mouse happened to land on the sidebar so then I was scrolling the long sidebar instead of the page. Also, the list of all charts seems somewhat less important to me than the first section in the entry, and expanded-by-default seems to be giving it more importance I think.

Anyway, will move onto code review now.

File: site/blocks/RelatedCharts.tsx [hunk 1]
```diff
@@ -48,7 +49,15 @@ export const RelatedCharts = ({ charts }: { charts: RelatedChart[] }) => {
...
                                     }}
                                 >
-                                    {chart.title}
+                                    <img
+                                        src={`${BAKED_GRAPHER_EXPORTS_BASE_URL}/${chart.slug}.svg`}
+                                        loading="lazy"
+                                        data-no-lightbox
+                                        data-no-img-formatting
+                                        width={850}
+                                        height={600}
```
Comment (danielgavrilov) [inline site/blocks/RelatedCharts.tsx:58]: I wonder if we should extract these into `GrapherConstants.ts` so they are synchronised between site and grapher?

**Comment by marcelgerber** [comment]
> The active state was confusing for me – I was wondering why the first chart is disabled. Instead of de-emphasising, maybe it's better to emphasise the selection? I think it's what I was expecting.

I agree with gavrilov here. An emphasised selection (e.g. blue border, more opacity) would be better.

File: site/blocks/RelatedCharts.tsx (hunk 1 above)
Comment (mlbrgl) [reply site/blocks/RelatedCharts.tsx:58]: yep makes sense, thanks! I've replaced them in a couple more places in grapher code too.

**Comment by mlbrgl** [comment]
- [x] active state more prominent
- [x] refactor default chart bounds into variables

### Keep open state by default?
**Relevant charts are visible straightaway**:  can be either a plus or minus, depending on their relative importance compared to the first section of the page (see [@danielgavrilov's comment](https://github.com/owid/owid-grapher/pull/1414#issuecomment-1110923001))

**Why change now?**: longer chart lists used to require a page scroll. In the suggested implementation, this isn't the case anymore: the page can be scrolled independently of the list (although that might not be immediately obvious for some readers). Additionally, this toggle has been deemed not very discoverable in the past, so readers might go pass it without noticing the affordance.

If showing relevant charts first thing on the page, without further interaction, is desirable, then I would consider the UX trade-off acceptable. Otherwise, it might be better to keep it closed.

**Pull request merged**